driver-scala/src/it/scala/org/mongodb/scala/ClientSideEncryptionBypassAutoEncryptionSpec.scala

```diff
@@ -79,6 +79,8 @@ class ClientSideEncryptionBypassAutoEncryptionSpec extends RequiresMongoDBISpec
     val result = collection.find().first().head().futureValue
 
     result.get[BsonString]("encryptedField") should equal(Some(fieldValue))
+
+    clientEncrypted.close()
```

Comment:
There's probably a way to ensure that this happens even when an exception is thrown, but I suppose it's not so important.